openedx/features/partners/helpers.py

```diff
@@ -75,11 +73,11 @@ def get_course_description(course):
     return description
 
 
-def import_module_using_slug(partner_slug):
+def import_form_using_slug(partner_slug):
     try:
-        return import_module(PARTNERS_VIEW_FRMT.format(slug=partner_slug))
+        return import_module(PARTNERS_FORM_FRMT.format(slug=partner_slug))
     except ImportError:
-        raise Http404('Your partner is not properly registered')
+        return None
```

Comment:
lets return default form here?

Reply:
This would require me to separately handle both form names from the returned module where it's being used. I think importing a default form in the file itself and just placing a check for `None` works?